databases_shiny_apps/connect-access-db.R

```diff
@@ -0,0 +1,74 @@
+library(tidyverse) #This package helps import spreadsheet-formatted data, primarily from Excel
+# library(data.table) #Helps with data management
+# library(dbplyr) #This package will help interface with databases by translating R code into database-specific variants
```

Comment:
```suggestion
library(tidyverse)
# library(data.table)
# library(dbplyr)
```

These comments aren't very helpful, as if a reader doesn't know what a library does they should go look it up themeselves.